sys/net/application_layer/gcoap/gcoap.c

```diff
@@ -732,6 +881,95 @@ static void _find_obs_memo_resource(gcoap_observe_memo_t **memo,
     }
 }
 
+/*
+ * Transport layer functions
+ */
+
+static void _tl_get_socket(coap_socket_t *sock)
```

Comment:
```suggestion
static void _tl_init_socket(coap_socket_t *sock)
```

Reply:
Wouldn't `tl_init_coap_socket()` be more clear?

Reply:
Changed.